apps/designer/app/designer/features/sidebar-left/panels/asset-manager/asset-info.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  Box,
+  Flex,
+  Grid,
+  IconButton,
+  Text,
+  Separator,
+  Button,
+} from "@webstudio-is/design-system";
+import type { Asset } from "@webstudio-is/prisma-client";
+import { getFormattedAspectRatio } from "./utils";
+import {
+  CloudIcon,
+  AspectRatioIcon,
+  Cross1Icon,
+  SizeIcon,
+  TrashIcon,
+} from "@webstudio-is/icons";
+import { useSubmit } from "@remix-run/react";
+import prettyBytes from "pretty-bytes";
+
+const truncatedText = {
+  position: "relative",
+  maxWidth: 80,
+
+  div: {
+    whiteSpace: "nowrap",
+    textOverflow: "ellipsis",
+    overflow: "hidden",
+
+    "&::after": {
+      content: "attr(data-extension)",
+      position: "absolute",
+      left: "100%",
+      top: 0,
+      whiteSpace: "nowrap",
+    },
+  },
+};
+
+type AssetInfoProps = Asset & {
+  onClose: () => void;
+  setIsDeleting: (isDeleting: boolean) => void;
+};
+
+export const AssetInfo = ({
+  size,
+  width,
+  height,
+  id,
+  name,
+  onClose,
+  setIsDeleting,
+}: AssetInfoProps) => {
+  const submit = useSubmit();
+  const handleDeleteAsset = () => {
+    const formData = new FormData();
+    formData.append("assetId", id);
+    formData.append("assetName", name);
+    onClose();
+    setIsDeleting(true);
+    submit(formData, { method: "delete" });
+  };
+
+  return (
+    <>
+      <Flex
+        css={{ height: 40, paddingLeft: "$3" }}
+        align="center"
+        justify="between"
+      >
+        <Text size="1" weight="700">
```

Comment:
is it possible that we could reuse the Header that we currently have in sidebar-left/lib/header? I haven't checked if there are any differences in styling

Reply:
It does work, changed to it